### DISCUSSION TITLE
Geoserver 6-1

### DISCUSSION THREAD

**Discussion by kathrynbond** [ua-gist604b-s24/syllabus#27] [General]
Having trouble with 6-1. I've watched the new tutorial a few times, but when I get to the point of running docker compose up -d, I get an error that states "docker daemon is not running." I have docker desktop open. I have restarted it and reinstalled docker. Any ideas I'm stuck.

![Docker_daemon](https://github.com/ua-gist604b-s24/syllabus/assets/152413661/43f3d659-a7ae-47c9-98ce-9f98ff75ce20)

**Comment by aaryno** [comment]
Can you share the output of `docker info`?

**Reply by kathrynbond** [reply]
![Docker_daemon](https://github.com/ua-gist604b-s24/syllabus/assets/152413661/dc836979-64c9-4e8d-91be-ab89f6f2a0d7)

**Comment by aaryno** [comment]
I found this, which indicates you need to enable WSL: https://forums.docker.com/t/error-response-from-daemon-open-pipe-docker-engine-windows-the-system-cannot-find-the-file-specified/131750

There are a few suggestions there.
Several also suggest typing this in Powershell (and rebooting):
```
Enable-WindowsOptionalFeature -Online -FeatureName $(“Microsoft-Hyper-V”, “Containers”) -All 
```

**Comment by kathrynbond** [comment]
So it turns out it must have been my security protocols at work because I've made it to my home computer, and it works now.
